docs/example_ScalarField_Geotiff_Bigger.html

```diff
@@ -38,7 +38,7 @@ <h1 class="title mapTitle">ScalarField GeoTIFF</h1>
         */
         d3.request("data/ndvi_201702_LZW.tif").responseType('arraybuffer').get(
             function (error, tiffData) {
-                let ndvi = L.ScalarField.fromGeoTIFF(tiffData.response);
+                let ndvi = L.ScalarField.fromGeoTIFF(tiffData.response, [0])[0];
```

Comment:
We will avoid this with the new method. This is an example of breaking an existing client

Reply:
Fixed